client/lib/performance-tracking/components/performance-tracker-stop.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { isSingleUserSite, isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import isSiteStore from 'state/selectors/is-site-store';
+import { stopNavigation } from '../api';
+
+class PerformanceTrackerStop extends React.Component {
+	static propTypes = {
+		// Indicates if the site is a Single site
+		single: PropTypes.bool,
+
+		// Indicates if the site is Atomic
+		at: PropTypes.bool,
+
+		// Indicates if the site has Jetpack
+		jetpack: PropTypes.bool,
+
+		// Indicates if the site has WooCommerce
+		store: PropTypes.bool,
+	};
```

Comment:
Are these also required?

Reply:
As far as I know they can be null if the store doesn't have that info yet.